references/obj_detection/train_pytorch.py

```diff
@@ -232,8 +233,11 @@ def main(args):
     train_set = DocArtefacts(
         train=True,
         download=True,
-        img_transforms=T.Resize((args.input_size, args.input_size)),
-    )
+        img_transforms=Compose([
+            T.Resize((args.input_size, args.input_size)),
+            ColorJitter(brightness=0.3, contrast=0.3, saturation=0.3, hue=0.02),
+            T.RandomApply(GaussianBlur(kernel_size=(3, 3), sigma=(0.1, 3)), .3)
```

Comment:
mind adding a comma at the end of the line please?

Reply:
Yes, thanks!